lms/templates/fields/field_textarea.underscore

```diff
@@ -1,6 +1,10 @@
 <div class="wrapper-u-field">
     <div class="u-field-header">
-        <label class="u-field-title" for="u-field-textarea-<%- id %>" id="u-field-title-<%- id %>" aria-describedby="u-field-message-help-<%- id %>"></label>
+        <% if (mode === 'edit') { %>
+            <label class="u-field-title" for="u-field-textarea-<%- id %>" id="u-field-title-<%- id %>" aria-describedby="u-field-message-help-<%- id %>"></label>
+        <% } else { %>
+            <span class="u-field-title" id="u-field-title-<%- id %>" aria-describedby="u-field-message-help-<%- id %>"></span>
```

Comment:
I'm pretty sure the ```u-field-message-help-``` would never be populated here, so I would get rid of the aria-describedby. Voiceover has recently started making benign containers, like spans and divs, accessibility objects, like groups and regions, if there is an aria-attribute on them.  So this is getting announced as a group and its very confusing and verbose/chatty.